data/json/mutations/mutations.json

```
@@ -716,7 +716,7 @@
     "id": "STRICT_HUMANITARIAN",
     "name": { "str": "Strict Humanitarian" },
     "points": 1,
-    "description": "You'd never eat people, but these things that look vaguely human from elsewhere are definitely not people.",
+    "description": "Your culinary definitions were always quite relaxed. You no longer receive a mood malus from eating otherwordly humanoids.",
```

```suggestion
    "description": "You're convinced that these things that look vaguely human from elsewhere are definitely not people. Therefore it's okay to eat them, right?",
```
Please don't mention mood malus in the description, it breaks the immersion.

---

I considered removing the "mood malus" part, but other traits also clearly reference gameplay without breaking immersion.
Nevertheless, your description is about 50 times better than mine could ever be, so thanks for that.